src/main/java/nextstep/ladders/domain/Lines.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.ladders.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lines {
+
+    private final List<Line> lines;
+
+    public Lines(final int countOfPerson, final int height) {
+        RandomPointGenerateStrategy strategy = new RandomPointGenerateStrategy();
+        this.lines = IntStream.range(0, height)
+                .mapToObj(i -> new Line(countOfPerson, strategy))
```

Comment:
`Stream.generate`를 사용해도 된답니다!
```suggestion
        this.lines = Stream.generate(() -> new Line(countOfPerson, strategy))
                .limit(height)
```

Reply:
되게 좋은거같아요!! 감사합니다 😄